tests/fast_tests/test_rllib_examples.py

```diff
@@ -0,0 +1,73 @@
+import json
+import unittest
+import os
+
+os.environ["TEST_FLAG"] = "True"
+
+import ray
+import ray.rllib.ppo as ppo
+from ray.tune import run_experiments
+from ray.tune.registry import register_env
+
+from flow.utils.registry import make_create_env
+from flow.utils.rllib import FlowParamsEncoder
+
+# use this to specify the environment to run
+
+# number of rollouts per training iteration
+N_ROLLOUTS = 2
+# number of parallel workers
+PARALLEL_ROLLOUTS = 2
+
+class TestRllibBenchmarks(unittest.TestCase):
+    """Test that the benchmarks run"""
+    def test_grid(self):
+        from flow.benchmarks.grid0 import flow_params
+        # get the env name and a creator for the environment
+        create_env, env_name = make_create_env(params=flow_params, version=0)
+
+        # initialize a ray instance
+        ray.init(redirect_output=False)
+
+        horizon = 64
+        config = ppo.DEFAULT_CONFIG.copy()
+        config["num_workers"] = PARALLEL_ROLLOUTS
+        config["min_steps_per_task"] = horizon
+        config["timesteps_per_batch"] = horizon * N_ROLLOUTS
+        config["vf_loss_coeff"] = 1.0
+        config["kl_target"] = 0.02
+        config["use_gae"] = True
+        config["horizon"] = horizon
+        config["clip_param"] = 0.2
+        config["sgd_batchsize"] = horizon
+
+        # save the flow params for replay
```

Comment:
take this out from test. it won't fix it, but if this fails we'll be getting two tests failing instead of one.